client/state/happychat/middleware.js

```diff
@@ -178,24 +178,45 @@ const sendMessage = ( connection, message ) => {
 	connection.notTyping();
 };
 
-export const sendInfo = ( connection, { getState }, siteUrl ) => {
-	const siteHelp = `\nSite I need help with: ${ siteUrl }`;
-	const screenRes = ( typeof screen === 'object' ) && `\nScreen Resolution: ${ screen.width }x${ screen.height }`;
-	const browserSize = ( typeof window === 'object' ) && `\nBrowser Size: ${ window.innerWidth }x${ window.innerHeight }`;
-	const userAgent = ( typeof navigator === 'object' ) && `\nUser Agent: ${ navigator.userAgent }`;
-	const localDateTime = `\nLocal Date: ${ moment().format( 'h:mm:ss a, MMMM Do YYYY' ) }`;
-
-	// Geo location
+export const sendInfo = ( connection, { getState }, action ) => {
+	const { howCanWeHelp, howYouFeel, site } = action;
+	const info = {
+		howCanWeHelp,
+		howYouFeel,
+		site,
```

Comment:
I think site can be a pretty large object. Are we sure we want to send the entire thing and not just the info we need? If we just dispatch the site id we can then use the site selectors to stick the info we want into this `info` object.

Also on the happychat side we'd probably be sticking the entire site object into state which would accumulate more data than necessary and it could actually become completely stale if the user updates anything in calypso.